biogrid/libs/biogrid-simulator/src/lib/biobrain/biobrain.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { BioBrain } from "./";
+
+let actualBrain: BioBrain;
+
+beforeAll(() => {
+  actualBrain = BioBrain.Instance;
+});
+
+describe('BioBrain class', () => {
+  test('It should test that there is only one instance of brain in the system', () => {
+    const expectedBrain = BioBrain.Instance;
+
+    expect(actualBrain).toEqual(expectedBrain);
```

Comment:
I'm not sure this is testing what you're looking for?  You're checking that `BioBrain.Instance == BioBrain.Instance`, not that there's only 1

Reply:
> I'm not sure this is testing what you're looking for? You're checking that `BioBrain.Instance == BioBrain.Instance`, not that there's only 1

I will revise the test, so that I can include call functions of the brain, and they should return the same value even after there have have been changes made to one of the instances of the brain created in the app

Reply:
Implement by changing the states of the brain, The test passed indicating a singleton